play.pokemonshowdown.com/src/battle.ts

```diff
@@ -2229,6 +2229,20 @@ export class Battle {
 			let item = Dex.items.get(args[2]);
 			let effect = Dex.getEffect(kwArgs.from);
 			let ofpoke = this.getPokemon(kwArgs.of);
+			if (!poke) {
+				if (effect.id === 'frisk') {
+					const possibleTargets = ofpoke!.side.foe.active.filter(p => p !== null);
```

Comment:
Wouldn't it be easier to use `find` instead of `filter`?

Reply:
find doesn't account for having >1 possible target.